### PR TITLE
Added Tags Collection 130

### PR DESCRIPTION
**Pull request by jerud3v**
#### Fix `130`
 - Added Tags Collection
 - fixed `products` relation to `tags`

### CODE REVIEW

File: api/tag/models/tag.settings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "description": {
+      "type": "text",
+      "required": true
+    },
+    "products": {
+      "collection": "product",
+      "via": "tags",
+      "dominant": true
+    }
+  }
```
Comment (jerud3v) [inline api/tag/models/tag.settings.json:26]: for reviewer reference

File: api/product/models/product.settings.json [hunk 2]
```diff
@@ -79,13 +76,17 @@
...
     "categories": {
-      "via": "products",
       "collection": "category",
+      "via": "products",
       "dominant": true
+    },
+    "tags": {
+      "via": "products",
+      "collection": "tag"
     }
```
Comment (jerud3v) [inline api/product/models/product.settings.json:90]: for reviewer reference

File: api/tag/models/tag.settings.json (hunk 1 above)
Comment (kristher1619) [reply api/tag/models/tag.settings.json:26]: can we have slug field for url purpose

File: api/tag/models/tag.settings.json [hunk 3]
```diff
@@ -22,6 +22,11 @@
       "collection": "product",
       "via": "tags",
       "dominant": true
+    },
+    "slug": {
+      "type": "uid",
+      "targetField": "name",
+      "required": false
     }
```
Comment (jerud3v) [inline api/tag/models/tag.settings.json]: for reviewers reference

**Pull request merged**